specification/machinelearningservices/data-plane/Microsoft.MachineLearningServices/preview/2019-05-01/execution.json

```diff
@@ -0,0 +1,1103 @@
+          "enum": [
+            "Mount",
+            "Download",
+            "Upload"
+          ],
+          "type": "string",
+          "example": "Mount",
+          "x-ms-enum": {
+            "name": "DataStoreMode",
+            "modelAsString": true
+          }
+        },
+        "pathOnDataStore": {
+          "description": "Relative path on the datastore.",
+          "type": "string",
+          "example": "/images/validation"
+        },
+        "pathOnCompute": {
+          "description": "The path on the compute target.",
+          "type": "string"
+        },
+        "overwrite": {
+          "description": "Whether to overwrite the data if existing.",
+          "type": "boolean",
+          "example": "False"
+        }
+      }
+    },
+    "EnvironmentDefinition": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name of the environment.",
+          "type": "string",
+          "example": "mydevenvironment"
+        },
+        "version": {
+          "description": "The environment version.",
+          "type": "string",
+          "example": "1"
+        },
+        "python": {
+          "$ref": "#/definitions/PythonSection",
+          "description": "Settings for a Python environment."
+        },
+        "environmentVariables": {
+          "description": "Definition of environment variables to be defined in the environment.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "docker": {
+          "$ref": "#/definitions/DockerSection",
+          "description": "The definition of a Docker container."
+        },
+        "spark": {
+          "$ref": "#/definitions/SparkSection",
+          "description": "The configuration for a Spark environment."
+        },
+        "inferencingStackVersion": {
+          "description": "The Inferencing stack version.",
+          "type": "string",
+          "example": "latest"
+        }
+      }
+    },
+    "HistoryConfiguration": {
+      "type": "object",
+      "properties": {
+        "outputCollection": {
+          "description": "Set to true to collect outputs and store in run history.",
+          "type": "boolean",
+          "example": "True"
+        },
+        "directoriesToWatch": {
+          "description": "The list of directories to monitor and upload files from.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "example": "[\"logs\", \"outputs\"]"
+        }
+      },
+      "additionalProperties": {
+        "type": "object"
+      }
+    },
+    "SparkConfiguration": {
+      "type": "object",
+      "properties": {
+        "configuration": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "AMLComputeConfiguration": {
```

Comment:
We shouldn't document the autocluster feature -- it's a path we're moving away from.

Reply:
@wbaumann-microsoft can you explain what all definitions and api contribute to autocluster feature? I can mark the obsolete

Reply:
Just the AMLComputeConfiguration class and the property of that type.

Please don't mark it as Obsolete -- it's not (yet) deprecated, just something we're moving away from.

Reply:
done